modules/nf-core/merquryfk/merquryfk/main.nf

```diff
@@ -37,14 +45,18 @@ process MERQURYFK_MERQURYFK {
     }
     def args = task.ext.args ?: ''
     prefix = task.ext.prefix ?: "${meta.id}"
+    def mat_ktab = matktab ? "${matktab.find{ it.toString().endsWith(".ktab") }}" : ''
+    def pat_ktab = patktab ? "${patktab.find{ it.toString().endsWith(".ktab") }}" : ''
     def FASTK_VERSION = 'f18a4e6d2207539f7b84461daebc54530a9559b0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     def MERQURY_VERSION = '8ae344092df5dcaf83cfb7f90f662597a9b1fc61' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     """
     MerquryFK \\
         $args \\
         -T$task.cpus \\
         ${fastk_ktab.find{ it.toString().endsWith(".ktab") }} \\
-        $assembly \\
+        ${mat_ktab} \\
```

Comment:
Does the tool not need some kind of argument to know what file is what?

Reply:
the usuage suggests the two optional ktabs comes after the read ktab.